src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -191,6 +193,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                     return LanguageVersion.CSharp7_3;
 
                 // C# 7.2 features.
+                case MessageID.IDS_FeatureAsyncStreams: // PROTOTYPE(async-streams)
                 case MessageID.IDS_FeatureNonTrailingNamedArguments: // semantic check
```

Comment:
Should be 7.3?

Reply:
It will become 8.0. But since I started work in a branch with 7.2 only, I stuck with 7.2 with a PROTOTYPE.